.tekton/basic-component-pipeline.yaml

```
@@ -8,7 +8,7 @@ spec:
   - name: slack-notification
     params:
     - name: message
-      value: ':x: Build for <https://console.redhat.com/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/$(context.pipelineRun.name)|Konflux pipeline> (image: `$(params.output-image-repo)`, revision <https://github.com/stackrox/stackrox/$(params.revision)|$(params.revision)>) has failed.'
+      value: ':x: {{event_type}} pipeline for <https://console.redhat.com/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/$(context.pipelineRun.name)|$(context.pipelineRun.name)> (`$(params.output-image-repo)`, revision <$(params.git-url)/$(params.revision)|$(params.revision)>) has failed.'
```

[nit] Links lead to 404 without `/commit/` in the url (e.g. [this](https://github.com/stackrox/stackrox/438d6754b05713a452ed78922d2c35816e5acfa2) vs [this](https://github.com/stackrox/stackrox/commit/438d6754b05713a452ed78922d2c35816e5acfa2)).

```suggestion
      value: ':x: {{event_type}} pipeline for <https://console.redhat.com/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/$(context.pipelineRun.name)|$(context.pipelineRun.name)> (`$(params.output-image-repo)`, revision <$(params.git-url)/commit/$(params.revision)|$(params.revision)>) has failed.'
```

---

Done in [02f9f05](https://github.com/stackrox/stackrox/pull/12312/commits/02f9f058d5e5a2a14ea4246c19121cb1afdb9b2b)